apps/emqx/src/emqx.app.src

```diff
@@ -14,7 +14,6 @@
         esockd,
         cowboy,
         sasl,
-        os_mon,
```

Comment:
Remove this may cause this application absent from the release package, I am not sure.

Reply:
I't ok, `os_mon` has already added in the root source: `reboot_lists.term` file.
It will also add to the release.

if we put `os_mon` in emqx.app.src, it will start as permanent, It will crash emqx application(then the whole vm is crash)  when os_mon's crash.
So I start `os_mon` application as temporarily in `emqx_os_mon` process's init.

Reply:
Alternatively, it could be moved to `included_applications`, so it's loaded but not started by the application controller.